### PR TITLE
Update 1and1.es.xml

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
There are many more to add, here is a list that I roughly went through:

<details><summary>Click to expand</summary>

account.1and1.es
admin.1and1.es
analitica.1and1.es
ayuda.1and1.es
www.ayuda.1and1.es
m.ayuda.1and1.es
buscadores.1and1.es
clientes.1and1.es
contabilidad-online.1and1.es
contacto.1and1.es
contrasena.1and1.es
contrato.1and1.es
digital-success.1and1.es
domain-center.1and1.es
domains-center.1and1.es
email.1and1.es
www.email.1and1.es
exchange.1and1.es
xadmin.1.exchange.1and1.es
xadmin.exchange.1and1.es
ias.1and1.es
listingcheck.1and1.es
listlocal.1and1.es
listlocalcheck.1and1.es
location-marketing.1and1.es
login.1and1.es
mailxchange.1and1.es
www.mailxchange.1and1.es
mbe.1and1.es
mbe-agent.1and1.es
microshop.1and1.es
navigation.1and1.es
online-office.1and1.es
phpmyadmin.1and1.es
pixel.1und1.de
platform.1and1.es
postmaster-contact.1and1.es
ratings.1and1.es
redes-sociales.1and1.es
registrar.1and1.es
server-services.1and1.es
telesales-hosting.1and1.es
tienda-clientes.1and1.es
www.upselling-hosting.1and1.es
webanalytics.1and1.es
webmail.1and1.es
webmail2.1and1.es

</details>

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
Thanks @gloomy-ghost added

File: src/chrome/content/rules/1and1.es.xml [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+	<target host="location-marketing.1and1.es" />
+	<target host="login.1and1.es" />
+	<target host="mailxchange.1and1.es" />
+	<target host="www.mailxchange.1and1.es" />
+	<target host="mbe.1and1.es" />
+	<target host="mbe-agent.1and1.es" />
+	<target host="microshop.1and1.es" />
+	<target host="navigation.1and1.es" />
+	<target host="online-office.1and1.es" />
+	<target host="phpmyadmin.1and1.es" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/1and1.es.xml]: pixel.1und1.de

**Comment by gloomy-ghost** [comment]
Thanks!

**Pull request merged**